gradle/dependencies.gradle

```diff
@@ -10,7 +10,8 @@ ext {
   version = [
       mapboxMapSdk              : '9.1.0-SNAPSHOT',
       mapboxSdkServices         : '5.1.0-beta.1',
-      mapboxSdkDirectionsModels : '5.1.0-beta.1',
+      mapboxSdkServicesSnapshot : '5.1.0-20200217.082443-24',
+      mapboxSdkDirectionsModels : '5.1.0-20200217.082455-24',
```

Comment:
@LukasPaczos  @Guardiola31337  
is it ok, that `mapboxSdkServices` use a new  dependency `mapboxSdkServicesSnapshot : '5.1.0-20200217.082443-24'`

but `mapboxSdkGeoJSON ` and `mapboxSdkTurf` still use 
mapboxSdkServices         : '5.1.0-beta.1' . ?

or they all should use a new snapshot dependency?

Reply:
Ideally we should release a new beta of MAS including the latest changes. Could you run with that @korshaknn? cc @langsmith

Reply:
@Guardiola31337 haven't released it before, don't know the flow

Reply:
For now, we could also add `5.1.0-SNAPSHOT` to both `mapboxSdkDirectionsModels` and `mapboxSdkServices` versions.

Reply:
@LukasPaczos fixed